include/openenclave/internal/raise.h

```diff
@@ -67,15 +67,15 @@
 
 OE_EXTERNC_BEGIN
 
-#define OE_RAISE(RESULT, ...)                                    \
-    do                                                           \
-    {                                                            \
-        result = (RESULT);                                       \
-        if (result != OE_OK)                                     \
-        {                                                        \
-            OE_TRACE_ERROR("Failed::%s", oe_result_str(result)); \
-        }                                                        \
-        goto done;                                               \
+#define OE_RAISE(RESULT, ...)                             \
+    do                                                    \
+    {                                                     \
+        result = (RESULT);                                \
+        if (result != OE_OK)                              \
+        {                                                 \
+            OE_TRACE_ERROR(":%s", oe_result_str(result)); \
```

Comment:
Did you mean to leave a colon in the string here?  ":%s" as opposed to "%s"?

Reply:
Having ":" there was on purpose.